script.js

```diff
@@ -14,6 +14,81 @@ function hide() {
+    link: 'link_to_live_version.com',
+  },
+
+  dataDashboard: {
+    title: 'Data Dashboard Healthcare',
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    image: './images/Snapshoot Portfolio.jpg',
+    paragraph:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    link: 'link_to_live_version.com',
+  },
+
+  websitePortfolio: {
+    title: 'Website Portfolio',
+    technologies: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+    image: './images/Snapshoot Portfolio.jpg',
+    paragraph:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    link: 'link_to_live_version.com',
+  },
+};
+
+function detailsPopup(projectId) {
+  const container = document.querySelector('.project_list');
+  const details = document.createElement('div');
+  details.id = 'pop-up-window';
+  let listItems = `<li><p class='new-section-technology'>${projects[projectId].technologies[0]}</p></li><li><p class='new-section-technology'>${projects[projectId].technologies[1]}</p></li><li><p class='new-section-technology'>${projects[projectId].technologies[2]}</p></li>`;
+  if (projects[projectId].technologies.length > 3) {
+    listItems += `<li><p class='new-section-technology'>${projects[projectId].technologies[3]}</p></li>`;
+  }
+  details.innerHTML = `<div class='pop-up-container'>
+                          <h3 class='hdr' style='margin: 0;'>${projects[projectId].title}
+                            <a id='closePopUp' class='close-pop-up' onclick='closeDetailsPopup()'>
+                              <img src='./images/Icon - Cancel.svg' alt='Close menu'>
+                            </a>
+                          </h3>
+                          <ul class='lis'>${listItems}</ul>
+                          <img src='${projects[projectId].image}' alt='${projects[projectId].title}' class='pop-up-image'/>
+                          <div class='text-and-buttons'><p class='para'>${projects[projectId].paragraph}</p>
+                          <div id='detail-buttons'>
+                          <a class="button detl-butn" href='#'>See Live<img src='./images/Icon - Export.svg' alt='See live'></a>
+                          <a class="button detl-butn" href='#'>See Source<img src='./images/see-project.svg' alt='See source'></a>
+                          </div>
+                          </div>
+                          </div>`;
```

Comment:
- [ ] @team great work on your popup however on mobile view the top and bottom margins are small, kindly consider increasing the size, also the position of the close button is a bit lower than in the design, kindly check the comparison below for better understanding

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/233770571-c7ddddf7-9f4e-4a03-8121-83c4046d2af1.png) |  ![image](https://user-images.githubusercontent.com/61408860/233770616-fc1e5e38-a90a-4a14-bc3f-5111309a345a.png)

Reply:
- [ ] @team great work on your design, however, there is a small design flow in your desktop version, there is a big empty space at the bottom of the popup, I would recommend you consider removing it, below is a comparison for better understanding 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/233770762-355b8674-bc5f-4978-a30e-180c1521ff15.png) | ![image](https://user-images.githubusercontent.com/61408860/233770793-5dec2739-e4ec-4128-bb60-aed7a8675bfb.png)

Reply:
- [ ] As requested by the previous reviewer, great job fixing the desktop version, however the bottom buttons for the mobile version have now disappeared. Kindly fix this.

Your design
![image](https://user-images.githubusercontent.com/81932478/233781436-76aca5cf-3c46-4b11-831d-f7219ab0d583.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/233781449-33d20d0d-4464-4ccf-8c8b-f184445cf99f.png)